releases.md

```diff
@@ -1,8 +1,41 @@
+## v2.5.0
+
+Date: 2021-10-14 <br>
+Contract: fio.reqobt <br>
+Hash: 49bc74038797d7d482914d8934ed73dfac7fde548fda2bfe9e071aa81878db48 <br>
+CDT: fio.cdt (1.5.x) <br>
+Git Commit: 45140b11b370d0a5a107b64c917db23c22125e3c
+
+Date: 2021-10-14 <br>
+Contract: fio.system <br>
+Hash: 06a0146a702552e9957c0f21f9cd7f71602872f0a34cf98d65ddfda5e10d2711 <br>
+CDT: fio.cdt (1.5.x) <br>
+Git Commit: 45140b11b370d0a5a107b64c917db23c22125e3c
+
+Date: 2021-10-14 <br>
+Contract: fio.token <br>
+Hash: 0c997d252406dbef152c3ba38378462aad4e090fb81334fd4c496f8a568ce66d <br>
+CDT: fio.cdt (1.5.x) <br>
+Git Commit: 45140b11b370d0a5a107b64c917db23c22125e3c
+
+Date: 2021-10-14 <br>
+Contract: fio.treasury  <br>
+Hash: 0c997d252406dbef152c3ba38378462aad4e090fb81334fd4c496f8a568ce66d <br>
```

Comment:
This is the fio.token hash. I get:
b8831584d5e304d26518ca551fd8b474e3fd7daabd0b9acf24a0c8659803e942